server.go

```diff
@@ -82,6 +82,7 @@ type service struct {
 	server interface{} // the server for service methods
 	md     map[string]*MethodDesc
 	sd     map[string]*StreamDesc
+	meta   interface{}
```

Comment:
"metadata" as a member var means this is the metadata of this service. I believe nobody like inconsistent, long and inaccurate names. If you think "metadata" is not good, please make your proposal instead.